main/admin/settings.lib.php

```diff
@@ -173,6 +266,12 @@ function handlePlugins()
             $plugin_info = [];
             require $plugin_info_file;
 
+            $officialRibbon = '';
+            if (in_array($pluginName, $officialPlugins)) {
+                $officialRibbon = '<div class="ribbon-diagonal ribbon-diagonal-top-right ribbon-diagonal-official"><span>'.get_lang('PluginOfficial').'</span></div>';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 166 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5078559)